.vscode/launch.json

```diff
@@ -109,7 +109,11 @@
         "APP_BASE_PATH": "",
         "APP_BASE_URL": "http://localhost",
         "MENU_API_URL": "http://dev.amidostacks.com/api/menu",
-        "PACT_BROKER": "https://amido-stacks.pact.dius.com.au"
+        "PACT_BROKER": "https://amido-stacks.pact.dius.com.au",
+        "NEXT_PUBLIC_CONTENTFUL_SPACE_ID": "veka2or2977j",
+        "NEXT_PUBLIC_CONTENTFUL_ACCESS_TOKEN": "9L3ML2_zyQ2FrnFvuXeWES2wPWVqEnQj0Lx14UtAx4o",
+        "NEXT_PUBLIC_CONTENTFUL_PREVIEW_ACCESS_TOKEN": "4JrZBpD7-sDb_qjD4KB3Z34eNtCR1QqgvnCYlKqZ4Wg",
```

Comment:
Might need to rotate these and remove. If you need these locally, perhaps using a bashprofile?

Reply:
Removed